charts/rancher-opni/rancher-opni/0.1.2/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "rancher-opni.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "rancher-opni.labels" . | indent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "rancher-opni.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "rancher-opni.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/config.yaml") . | sha256sum }}
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+        name: kube-rbac-proxy
+        ports:
+        - containerPort: 8443
+          name: https
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        command:
+        - /manager
+        image: joekralicky/opni-manager:v0.1.1
```

Comment:
Should image be a value? also is it rancher/opni-manager now?

Reply:
Yeah good point, I'll update it.  We have the repo but haven't pushed anything to it yet.  Once https://github.com/rancher/opni/pull/66 is merged then new tags will get pushed to the repo